### PR TITLE
Add a SimpleLLM interface, and update documentation.

### PR DESCRIPTION
**Pull request by madiator**

### CODE REVIEW

File: .DS_Store
Comment (RyanMarten) [inline .DS_Store:1]: can you delete this file please?

File: docs/curator-dataset.png
Comment (RyanMarten) [inline docs/curator-dataset.png:1]: can you choose a more interesting screenshot than 1 request and 1 response?

File: README.md [hunk 1]
```diff
@@ -38,43 +38,83 @@ Bespoke Curator is an open-source project:
...
 class Poems(BaseModel):
     poems_list: List[Poem] = Field(description="A list of poems.")
+```
 
-
-# We define an `LLM` object that generates poems which gets applied to the topics dataset.
+We define an `LLM` object that generates poems which gets applied to the topics dataset.
+```python
 poet = curator.LLM(
     # `prompt_func` takes a row of the dataset as input.
```
Comment (RyanMarten) [inline README.md:119]: I find the comments here busy and confusing. 
put all before in a """ comment?

File: src/bespokelabs/curator/llm/llm.py [hunk 2]
```diff
@@ -37,79 +37,6 @@
 class LLM:
     """Interface for prompting LLMs."""
```
Comment (RyanMarten) [inline src/bespokelabs/curator/llm/llm.py:39]: I think that __init__ should be the first function in the class?

**Review by RyanMarten — APPROVED** [review]
some small comments

File: src/bespokelabs/curator/llm/llm.py (hunk 2 above)
Comment (madiator) [reply src/bespokelabs/curator/llm/llm.py:39]: Yes, absolutely! I was planning to mention this and move it, but for clarity thought i will remove the incorrect one and keep the correct one and then move it. Moved it now.

File: README.md (hunk 1 above)
Comment (madiator) [reply README.md:119]: How about now?

**Review by madiator — COMMENTED** [review]
addressed

File: docs/curator-dataset.png
Comment (madiator) [reply docs/curator-dataset.png:1]: done

**Pull request merged**